fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/SetMultiShardModifyModeAction.java

```diff
@@ -43,44 +41,13 @@ public SetMultiShardModifyModeAction(Action next) {
     @Override
     public void performOn(FHIRDbFlavor flavor, Connection connection) throws FHIRPersistenceDBConnectException {
 
-        if (flavor.getType() == DbType.CITUS) {
+        if (flavor.getType() == DbType.CITUS && USE_SEQUENTIAL) {
             // This is only used for Citus databases
-            log.fine("SET LOCAL citus.multi_shard_modify_mode TO 'sequential'");
             ConfigureConnectionDAO dao = new ConfigureConnectionDAO();
             dao.run(new CitusTranslator(), connection);
         }
 
         // perform next action in the chain
         super.performOn(flavor, connection);
     }
-
-    /**
-     * Create the declared global temporary table COMMON_TOKEN_VALUES_TMP
-     * @param adapter
-     * @throws FHIRPersistenceDBConnectException
-     */
-    public void createCommonTokenValuesTmp(DerbyAdapter adapter) throws FHIRPersistenceDBConnectException {
-        IDatabaseStatement cmd = new CreateCommonTokenValuesTmp();
-        adapter.runStatement(cmd);
-    }
-
-    /**
-     * Create the declared global temporary table CODE_SYSTEMS_TMP
-     * @param adapter
-     * @throws FHIRPersistenceDBConnectException
-     */
-    public void createCodeSystemsTmp(DerbyAdapter adapter) throws FHIRPersistenceDBConnectException {
-        IDatabaseStatement cmd = new CreateCodeSystemsTmp();
-        adapter.runStatement(cmd);
-    }
-
-    /**
-     * Create the declared global temporary table COMMON_TOKEN_VALUES_TMP
-     * @param adapter
-     * @throws FHIRPersistenceDBConnectException
-     */
-    public void createCanonicalValuesTmp(DerbyAdapter adapter) throws FHIRPersistenceDBConnectException {
-        IDatabaseStatement cmd = new CreateCanonicalValuesTmp();
-        adapter.runStatement(cmd);
-    }
```

Comment:
can we remove the underlying classes now?
CreateCommonTokenValuesTmp, CreateCodeSystemsTmp, and CreateCanonicalValuesTmp?